### ISSUE TITLE
[DAB] Add support for uGreen DABBOARD

### ISSUE THREAD

**Issue by rubenverhoef** [rubenverhoef/OAP-Config#17]
- make I2S work
- External applications to start radio stations
- buttons 0-9 to start radio stations (#5)

**Comment by EricJanVanDenBogaard** [comment]
Hebben jullie al hardware opties om de dabboard met knoppen, of rotary encoders te bedienen ?

**Comment by rubenverhoef** [comment]
Nee, de DAB board wordt vanuit [OAP](https://bluewavestudio.io/index.php/bluewave-shop/openauto-pro-detail) en dashboard knoppen (via CANbus) aangestuurd.
